### PR TITLE
Prepare new release

### PR DESCRIPTION
**Pull request by lukasmatusiewicz**
- [x] Update pom.xml
- [x] Update the java-client
- [x] Update the changelog

### CODE REVIEW

File: src/main/java/org/privacyidea/authenticator/PrivacyIDEAAuthenticator.java [hunk 1]
```diff
@@ -517,11 +578,18 @@ private Map<String, String> getHeadersToForward(AuthenticationFlowContext contex
...
+        for (String header : config.forwardedHeaders()
+                                   .stream()
+                                   .distinct()
+                                   .collect(Collectors.toList()))
         {
-            List<String> headerValues = context.getSession().getContext().getRequestHeaders().getRequestHeaders()
+            List<String> headerValues = context.getSession()
+                                               .getContext()
+                                               .getRequestHeaders()
+                                               .getRequestHeaders()
```
Comment (nilsbehlen) [inline src/main/java/org/privacyidea/authenticator/PrivacyIDEAAuthenticator.java:592]: ` .getRequestHeaders()` twice?

File: src/main/java/org/privacyidea/authenticator/PrivacyIDEAAuthenticator.java [hunk 2]
```diff
@@ -452,37 +502,45 @@ private void extractChallengeDataToForm(PIResponse response, AuthenticationFlowC
...
-            context.form().setAttribute(FORM_PUSH_MESSAGE, response.pushMessage());
+            context.form()
+                   .setAttribute(FORM_PUSH_AVAILABLE, true);
+            context.form()
+                   .setAttribute(FORM_PUSH_MESSAGE, response.pushMessage());
         }
 
         // Check for WebAuthn and U2F
-        if (response.triggeredTokenTypes().contains(TOKEN_TYPE_WEBAUTHN))
+        if (response.triggeredTokenTypes()
```
Comment (nilsbehlen) [inline src/main/java/org/privacyidea/authenticator/PrivacyIDEAAuthenticator.java:530]: these should be on one line

File: src/main/java/org/privacyidea/authenticator/PrivacyIDEAAuthenticator.java [hunk 3]
```diff
@@ -186,12 +207,19 @@ else if (!config.excludedGroups().isEmpty())
...
+               .setAttribute(FORM_WEBAUTHN_SIGN_REQUEST, "")
+               .setAttribute(FORM_U2F_SIGN_REQUEST, "")
+               .setAttribute(FORM_PUSH_MESSAGE, pushMessage)
+               .setAttribute(FORM_OTP_AVAILABLE, true)
+               .setAttribute(FORM_OTP_MESSAGE, otpMessage)
+               .setAttribute(FORM_PUSH_AVAILABLE, false)
+               .setAttribute(FORM_IMAGE_PUSH, "")
+               .setAttribute(FORM_IMAGE_OTP, "")
+               .setAttribute(FORM_IMAGE_WEBAUTHN, "")
+               .setAttribute(FORM_POLL_INTERVAL, config.pollingInterval()
```
Comment (nilsbehlen) [inline src/main/java/org/privacyidea/authenticator/PrivacyIDEAAuthenticator.java:221]: `config.pollingInterval().get(0)` should be on one line

**Review by nilsbehlen — CHANGES_REQUESTED** [review]

File: src/main/java/org/privacyidea/authenticator/PrivacyIDEAAuthenticator.java (hunk 1 above)
Comment (lukasmatusiewicz) [reply src/main/java/org/privacyidea/authenticator/PrivacyIDEAAuthenticator.java:592]: yes

File: src/main/java/org/privacyidea/authenticator/PrivacyIDEAAuthenticator.java (hunk 3 above)
Comment (lukasmatusiewicz) [reply src/main/java/org/privacyidea/authenticator/PrivacyIDEAAuthenticator.java]: [6dfff71](https://github.com/privacyidea/keycloak-provider/pull/128/commits/6dfff7181a94790e4e689807c77c9af8107882a5)

File: src/main/java/org/privacyidea/authenticator/PrivacyIDEAAuthenticator.java (hunk 2 above)
Comment (lukasmatusiewicz) [reply src/main/java/org/privacyidea/authenticator/PrivacyIDEAAuthenticator.java]: [6dfff71](https://github.com/privacyidea/keycloak-provider/pull/128/commits/6dfff7181a94790e4e689807c77c9af8107882a5)

**Pull request merged**